README.md

```diff
@@ -242,3 +242,28 @@ The stop_delays view looks like this:
 ### Demo Project
 
 See the [gtfsrdb-delay-demo](https://github.com/CUTR-at-USF/gtfsrdb-delay-demo) project for a sample web application that visualizes these delays.
+
+### Scripts
+
+The scripts/ directory is provided as a "quick start" method for using GTFSrDB. A database server is still a prerequisite, and the scripts assume you're using PostgreSQL.
+
+To get started with the scripts, first make sure you have a good working Python environment. This involves setting up a virtual environment, then installing the required libraries. A quick way to do this is as follows:
+1. Ensure you are in the root directory of gtfsrdb on your PC.
+2. Create a Python virtual environment: `python3 -m venv venv`
+3. Install the required libraries: `pip3 install -r requirements.txt`
```

Comment:
There is no requirements.txt anymore. I defined the dependencies in the pyproject.toml which makes the requirements.txt obsolete. Are you fine with switching to it?
```suggestion
3. Install the package and the required libraries: `pip install -e .`
```